pkg/controllers/job/helpers/helpers.go

```diff
@@ -121,22 +121,22 @@ func GetJobKeyByReq(req *apis.Request) string {
 }
 
 // GetTasklndexUnderJob return index of the task in the job.
-func GetTasklndexUnderJob(taskName string, job *batch.Job) int {
+func GetTasklndexUnderJob(specName string, job *batch.Job) int {
```

Comment:
GetTasklndexUnderJob function is used to find task index in job. The `taskName`  is more straightforward and generic.